datafusion/src/physical_plan/hash_join.rs

```diff
@@ -1375,7 +1344,7 @@ mod tests {
             "| 1  | 4  | 7  | 10 | 4  | 70 |",
             "| 2  | 5  | 8  | 20 | 5  | 80 |",
             "| 2  | 5  | 8  | 20 | 5  | 80 |",
-            "| 3  | 7  | 9  |    | 7  |    |",
+            "| 3  | 7  | 9  |    |    |    |",
```

Comment:
I double checked this and the answer after this PR appears to be correct (there is no value `7` in `b1` on the right input 👍